pkg/volume/volume_info_common.go

```diff
@@ -0,0 +1,136 @@
+/*
+Copyright 2018 the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package volume
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+type VolumeInfoVersion struct {
+	Version string `json:"version"`
+}
+
+type VolumeInfos struct {
+	VolumeInfos []VolumeInfo `json:"volumeInfos"`
+}
+
+type VolumeInfo struct {
+	// The PVC's name.
+	PVCName string `json:"pvcName"`
+
+	// The PVC's namespace
+	PVCNamespace string `json:"pvcNamespace"`
+
+	// The PV name.
+	PVName string `json:"pvName"`
+
+	// The way the volume data is backed up. The valid value includes `VeleroNativeSnapshot`, `PodVolumeBackup` and `CSISnapshot`.
```

Comment:
Let's define some constants for these methods

Reply:
Added.